src/components/CreateBeer.js

```diff
@@ -0,0 +1,110 @@
+/** @format */
+
+import axios from "axios";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default function CreateBeer() {
+	const [name, setName] = useState("");
+	const [tagline, setTagline] = useState("");
+	const [description, setDescription] = useState("");
+	const [first_brewed, setFirst_brewed] = useState("");
+	const [brewers_tips, setBrewers_tips] = useState("");
+	const [contributed_by, setContributed_by] = useState("");
+	const [attenuation_level, setAttenuation_level] = useState(0);
+
+	const navigate = useNavigate();
+
+	const createNewBeer = (e) => {
+		e.preventDefault();
+		axios
+			.post(`https://ih-beers-api2.herokuapp.com/beers/new`, {
+				name,
+				tagline,
+				description,
+				first_brewed,
+				brewers_tips,
+				contributed_by,
+				attenuation_level,
+			})
+			.then((res) => {
+				console.log("created the...", res.data);
+				navigate("/beers");
+			})
+			.catch((e) => {
+				console.log(e);
+			});
+	};
+
+	return (
+		<div className="beer-create">
+			<h1>New Choice</h1>
+			<form onSubmit={(e) => createNewBeer(e)}>
+				<label>
+					Name
```

Comment:
I would also add something to indicate for the user what is mandatory to create a new thing :) 

```suggestion
					Name (*)
```